src/textual/widgets/_list_view.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import annotations
+
+import asyncio
+from asyncio import Future
+
+from textual import events
+from textual.binding import Binding
+from textual.containers import Vertical
+from textual.geometry import clamp
+from textual.message import Message
+from textual.reactive import reactive
+from textual.widgets._list_item import ListItem
+
+
+class ListView(Vertical, can_focus=True, can_focus_children=False):
+    DEFAULT_CSS = """
+    ListView {
+        scrollbar-size-vertical: 1;
```

Comment:
Subjective, but I prefer width 1 scrollbars for scrollbars that aren't Screen-level.